DCF_Demo.md

```diff
@@ -249,35 +287,36 @@ The commands are different for Powershell because you have to use ` to escape an
 
 Query for existing interfaces on the device 
 ```
-az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ipc.query.1.query" --mp "{}"
+az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ipc.*.query.1:query" --mp "{}"
 
 // expected outcome
 {
   "payload": {
     "continuation_token": 655615,
     "result": [
-      "+ipc.1.query.1",
-      "+ipc.1.interface_manager.1",
-      "+dm.1.packages.1"
+      "*ipc.1.query.1",
+      "*ipc.1.interface_manager.1",
+      "*dm.1.packages.1"
     ]
   },
   "status": 200
 }
 
 ```
 
+**Note**: The `*` in front of the names represent that this is default package for that interface.
```

Comment:
nit: please change all of these as per line #188

Reply:
#Pending